src/src/components/ingest/publications_edit.jsx

```diff
@@ -1866,7 +1866,7 @@ class PublicationEdit extends Component {
           key={idstr}
           id={idstr}
           onChange={this.handleInputChange}
-          checked={this.state.data_types.has(val.name)}
+          checked={this.state.dataset_type.has(val.name)}
```

Comment:
Will this still work when `dataset_type` is a string?

Reply:
the "renderOneAssay" function this is within is never used, and is leftover from copying from the Dataset form